plugins/reach/lib/api/data/entryVendorTaks/KalturaModerationVendorTaskData.php

```diff
@@ -50,4 +58,28 @@ public function toObject($dbObject = null, $propsToSkip = array())
 
 		return parent::toObject($dbObject, $propsToSkip);
 	}
+
+	public function validateForInsert($propertiesToSkip = array())
+	{
+		if($this->categoryIds)
+		{
+			$this->validateCategories();
+		}
+		return parent::validateForInsert($propertiesToSkip);
+	}
+
+	protected function validateCategories()
+	{
+		$categoryIds = explode(',', $this->categoryIds);
+
+		foreach ($categoryIds as $categoryId) {
```

Comment:
Move { to a new line
also in the next 'if's

Reply:
Done. tnx